Tests/RxSwiftTests/PrimitiveSequence+ConcurrencyTests.swift

```diff
@@ -41,6 +41,37 @@ extension PrimitiveSequenceConcurrencyTests {
             XCTAssertTrue(true)
         }
     }
+
+    func testSingleThrowsCancellationErrorWhenParentTaskIsCancelledButNoEventIsProduced() async throws {
```

Comment:
Lets shorten to something like this across the board

```suggestion
    func testSingleThrowsCancelleationWithoutEvents() async throws {
```